src/less/general.less

```diff
@@ -20,9 +20,20 @@ html.ext-wwt-popup body {
 
 .mw-parser-output .editor-token {
 	cursor: pointer;
+	position: relative;
+	display: inline-block;
```

Comment:
A little worried about setting all tokens as `inline-block` rather than `inline`, but if you don't see any negative impact, it's probably fine...? :D

Reply:
Below is what I've learned during my research :)

We need the inline element to behave as a block so that when an element starts with a space it renders in the same line.  Inline elements that start with a space and are at the end of a line, will start at the end of the line and then continue to the beginning of the next line.  The `::before` pseudo element with the background color will not follow the `inline` element, that they are attached to, to the next line. Here is a visual of the different types of display:  https://stackoverflow.com/questions/9189810/css-display-inline-vs-inline-block 
We are also able to add CSS attributes, like padding, for `inline-block` to cover the white lines between lines in paragraphs.

Do you foresee any possible issues with this solution?

Reply:
I think one of the issues with `inline-block` is that the whole thing will be wrapped to the next line sometimes, instead of broken in the middle. For example: https://codepen.io/freosam/pen/MWgxMPG This doesn't seem to be that much of an issue because most `editor-token` elements are next to others, but if they are next to e.g. an inline element introduced by a template then wrapping might go awry.

Reply:
I'm good with iterating on this; if it works 99% of the time and the 1% edge case is super edgy and probably not going to be introduced, then I think that's fine. Especially since this fixes a bigger edge case, so it's an overall win.

We can start with this potential problem and if we see that it appears often, we can rethink this piece, it's fairly contained so it's not like we will have a lot of trouble iterating on it.